dotCMS/src/main/java/com/dotcms/csspreproc/DotCSSCompiler.java

```diff
@@ -234,6 +237,10 @@ protected File createCompileDir(Host currentHost, String uri, boolean live)
     return compDir;
   }
 
+  private boolean areSiblings(final String uri, final String otherUri) {
+    return uri.substring(0, uri.lastIndexOf("/")).equals(otherUri.substring(0, otherUri.lastIndexOf("/")));
```

Comment:
not sure if this method should be promoted to some Util
Also use StringPool.SLASH

Reply:
It totally can.